UniConnectApp/app/src/main/java/com/example/uniconnect/dto/University.kt

```diff
@@ -2,7 +2,11 @@ package com.example.uniconnect.dto
 
 import com.google.gson.annotations.SerializedName
 
-data class University(@SerializedName("name") var name: String, @SerializedName("country")var country: String, @SerializedName("alpha_two_code") var code: String) {
+data class University(
+    @SerializedName("name") var name: String,
+    @SerializedName("country") var country: String,
+    @SerializedName("alpha_two_code") var code: String
+) {
```

Comment:
automatic via kotlin style guide per https://kotlinlang.org/docs/coding-conventions.html#apply-the-style-guide